### ISSUE TITLE
experiencing issues with installation on Ubuntu 14.04 (as advertised)

### ISSUE THREAD

**Issue by v1k0d3n** [SUSE/machinery#2085]
Ran into issues with running on Ubuntu:

```
root@machine01:~# gem install machinery-tool
Building native extensions.  This could take a while...
Successfully installed machinery-tool-1.20.0
1 gem installed
Installing ri documentation for machinery-tool-1.20.0...
Installing RDoc documentation for machinery-tool-1.20.0...
root@machine01:~# machinery help
/var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/machinery.rb:94:in `require_relative': /var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/file_validator.rb:70: syntax error, unexpected tLPAREN_ARG, expecting keyword_do_LAMBDA or tLAMBEG (SyntaxError)
    changes_proc = -> (file) { file["changes"] }
                       ^
/var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/file_validator.rb:70: syntax error, unexpected tLAMBEG, expecting keyword_end
    changes_proc = -> (file) { file["changes"] }
                              ^
/var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/file_validator.rb:70: syntax error, unexpected '}', expecting keyword_end
/var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/file_validator.rb:76: syntax error, unexpected tLPAREN_ARG, expecting keyword_do_LAMBDA or tLAMBEG
      changes_proc = -> (file) { file["changes"]}
                         ^
/var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/file_validator.rb:76: syntax error, unexpected tLAMBEG, expecting keyword_end
      changes_proc = -> (file) { file["changes"]}
                                ^
/var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/file_validator.rb:76: syntax error, unexpected '}', expecting keyword_end
	from /var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/lib/machinery.rb:94:in `<top (required)>'
	from /var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/bin/machinery:20:in `require_relative'
	from /var/lib/gems/1.9.1/gems/machinery-tool-1.20.0/bin/machinery:20:in `<top (required)>'
	from /usr/local/bin/machinery:23:in `load'
	from /usr/local/bin/machinery:23:in `<main>'
root@machine01:~#
```

**Comment by mauromorales** [comment]
Hi @v1k0d3n, thanks for submitting the issue. Machinery requires ruby 2.0 or higher and mistakenly it wasn't updated in the gemspec before. There already a new release in the oven which should see the light in the next couple of weeks hopefully but in the meantime if you could upgrade and test that would be great.

**Comment by fschueller** [comment]
Hi @v1k0d3n, this issue was fixed in v1.21.0 by requiring at least Ruby 2.0.
